examples/Python/ColorWheel/ColorWheel.pyde

```diff
@@ -0,0 +1,14 @@
+size(300, 300)
```

Comment:
Just noting here that ColorWheel is missing a header.pyde. I've already copied it over from ColorWheel.pde (and fixed the title there).

Reply:
thank you!